### PR TITLE
Tweak issue templates

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Implement new labels ("Package: Addition Request"; "Package: Removal Request"; "Package: Update Request")
- Unify style
- Unify wording
- Fix typo

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/request-package-removal.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Request a package to be removed
+description: Request removal of a package from the Solus repository. The package maybe unmaintained, replaced, deprecated or other reason.
```
Comment (EbonJaeger) [inline .github/ISSUE_TEMPLATE/request-package-removal.yml:3]: The second sentence is a bit awkward. Can we come up with better wording?

File: .github/ISSUE_TEMPLATE/request-package-removal.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+labels:
+  - Package: Removal Request
```
Comment (EbonJaeger) [inline .github/ISSUE_TEMPLATE/request-package-removal.yml:5]: Does this have to be in quotes?

File: .github/ISSUE_TEMPLATE/request-package-removal.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      label: Please confirm there isn't an open request for this package
+      description: Before opening a new issue, please search [open package removal requests](https://github.com/getsolus/packages/issues?q=sort%3Aupdated-desc+is%3Aopen+label%3A%22Package%3A+Removal+Request%22) to ensure there is not an existing one.
+      options:
+        - label: I have searched through package removal requests
+          required: true
+  - type: input
+    id: name
+    attributes:
+      label: Name
+      description: Name of the software.
```
Comment (EbonJaeger) [inline .github/ISSUE_TEMPLATE/request-package-removal.yml:19]: Would "Name of the package" be a better fit?

File: .github/ISSUE_TEMPLATE/request-package-update.yml [hunk 4]
```diff
@@ -1,14 +1,14 @@
 ---
-name: Package Update Request
-description: Request an update to a package available in the repository.
+name: Request a package to be updated
+description: Request an update to a package available in the Solus repository.
 labels:
-  - Package Update Request
+  - Package: Update Request
```
Comment (EbonJaeger) [inline .github/ISSUE_TEMPLATE/request-package-update.yml:5]: Does this have to be quoted, due to the `:`?

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
Overall good. I just have a few items.

File: .github/ISSUE_TEMPLATE/request-package-removal.yml (hunk 2 above)
Comment (malfisya) [reply .github/ISSUE_TEMPLATE/request-package-removal.yml:5]: From their docs:
`labels` (Array or String): This issue will automatically receive these labels upon creation. Can be array of labels or comma-delimited string, e.g. "bug,needs-triage"

Probably needs quotes?

Comment (malfisya) [reply .github/ISSUE_TEMPLATE/request-package-removal.yml]: Just tested it [here](https://github.com/getsolus/packages/commit/422ae16986f7878b2ca9112314bd08c37f0cf0d7). It needs quotes.

File: .github/ISSUE_TEMPLATE/request-package-removal.yml [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
+name: Request a package to be removed
+description: Request a package to be removed from the Solus repository. The package maybe unmaintained, replaced, deprecated or other reason.
```
Comment (EbonJaeger) [inline .github/ISSUE_TEMPLATE/request-package-removal.yml:3]: For the second sentence, maybe something like:

"The software may be unmaintained, replaced by another project, deprecated, or should be removed for some other reason."

**Review by EbonJaeger — CHANGES_REQUESTED** [review]

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks!

**Pull request merged**